core/app/models/spree/order.rb

```diff
@@ -21,6 +21,7 @@ class Order < Spree::Base
 
     alias display_ship_total display_shipment_total
     alias_attribute :ship_total, :shipment_total
+    alias_attribute :token, :guest_token # FIXME: guest_token should be renamed to just token
```

Comment:
This should be done on Serializer level. You have example of passing block to attribute:
https://github.com/Netflix/fast_jsonapi#attributes